### ISSUE TITLE
`make docker-build` failed because of get-pip Python 3.6 unsupport and apex build failure

### ISSUE THREAD

**Issue by y-okumura-isp** [TRI-ML/packnet-sfm#216]
When running `make docker-build`, the following error occurs.

(1) pip no more supports Python 3.6

It looks get-pip.py is updated on 03-Feb-2022 09:45.

```
ERROR: This script does not work on Python 3.6 The minimum supported Python version is 3.7. Please use https://bootstrap.pypa.io/pip/3.6/get-pip.py instead.
The command '/bin/bash -cu curl -O https://bootstrap.pypa.io/get-pip.py &&     python get-pip.py &&     rm get-pip.py' returned a non-zero code: 1
Makefile:65: recipe for target 'docker-build' failed
make: *** [docker-build] Error 1
```

(2) NVIDIA apex build failure

I got `ATen/cuda/DeviceUtils.cuh: No such file or directory`.
It looks [build Apex latest version failed with pytorch 1.4.0 due to missing ATen/cuda/DeviceUtils.cuh](https://github.com/NVIDIA/apex/issues/1200)

**Comment by xiaojiejieshiwode** [comment]
Hello, I have the same problem as you. Have you solved it？

**Comment by y-okumura-isp** [comment]
Yes, maybe. Though I don't know it is a right way, I could build by the following patch https://github.com/TRI-ML/packnet-sfm/pull/217

**Comment by markus-42** [comment]
I fixed this by replacing in 'docker/Dockerfile' line 78, which is

`RUN curl -O https://bootstrap.pypa.io/get-pip.py && \`

with the following:

`RUN curl -O https://bootstrap.pypa.io/pip/3.6/get-pip.py && \`
